src/Articles/components/AllArticles.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React, { Component } from "react";
+import { Container } from "react-bootstrap";
+import PropTypes from "prop-types";
+import Article from "../../commons/components/Article";
+import { connect } from "react-redux";
+import getAllArticles from "../redux/actions/AllArticlesActions";
+import '../css/AllArticles.scss';
```

Comment:
@neelxie please consider changing the css folder name to uppercase like `CSS` to avoid rebase issues.

Reply:
@alexxsanya thank you I have changed that.